src/Integration.UnitTests/Suppression/SuppressedIssuesProviderTests.cs

```diff
@@ -107,13 +107,43 @@ public void GetSuppressedIssues_HasSolutionBinding_ConnectedMode_ListFromBinding
             var bindingConfiguration = new BindingConfiguration(new BoundSonarQubeProject(), mode);
             var expectedIssues = SetupExpectedIssues(bindingConfiguration);
 
-            SetupBindingEvent(eventType, bindingConfiguration);
+            SimulateBindingEvent(eventType, bindingConfiguration);
 
             var actual = testSubject.GetSuppressedIssues("project guid", "file path");
 
             actual.Should().BeEquivalentTo(expectedIssues);
         }
 
+        [DataTestMethod]
+        [DataRow(SolutionBindingEventType.SolutionBindingUpdated)]
+        [DataRow(SolutionBindingEventType.SolutionBindingChanged)]
+        public void SolutionBindingEvent_Standalone_DoesNotRaiseSupressionsUpdatedEvent(SolutionBindingEventType eventType)
```

Comment:
FYI typo `...Supressions...` in this test and the next one. No need to fix it if you don't want to since it's in test code.

Reply:
I'll fix, ty!